packages/rs-dpp/src/identity/state_transition/identity_update_transition/validate_identity_update_transition_state.rs

```diff
@@ -147,6 +151,12 @@ where
         if !state_transition.get_public_keys_to_add().is_empty() {
             identity.add_public_keys(state_transition.get_public_keys_to_add().iter().cloned());
 
+            raw_public_keys = identity
```

Comment:
@antouhou fixed bug with missing identity public keys in the resulting vector that is about to be validated